gui/webdisplay/inc/ROOT/TWebWindowsManager.hxx

```diff
@@ -46,6 +46,9 @@ private:
    /// Release all references to specified window, called from TWebWindow destructor
    void Unregister(TWebWindow &win);
 
+   /// Provide URL address to access specified window from inside or from remote
```

Comment:
For functions that have an implementation in the source file, can you remove the `///` doc from headers?